plugins/generator-1.20.6/neoforge-1.20.6/mappings/enchantmenttypes.yaml

```diff
@@ -0,0 +1,58 @@
+_default: DIGGER

```

Comment:
![image](https://github.com/MCreator/MCreator/assets/16374228/b8298a3a-09c6-41b3-8f9c-b2775fb58d75)

Old mappings look quite different from what you have proposed here. If the system changed that much, it would make more sense to handle this together with enchantment ME